packages/net/tcp_connection.pony

```diff
@@ -187,7 +187,11 @@ actor TCPConnection
   var _listen: (TCPListener | None) = None
   var _notify: TCPConnectionNotify
   var _connect_count: U32
-  var _fd: U32 = -1
+  /* Docstring, waiting for PR ? """
+  Operating system file descriptor for this socket, -1 if
+  not open or connected.
+  """ */
+  var fd: U32 = -1
```

Comment:
why expost this?

Reply:
It's bullet point 3 in the description, sorry it isn't clearer.  Is there a better way to allow `TCPConnectionNotify.connected()` or `TCPListenNotify.connected()` or `.listening()` to access the socket's file descriptor?